vm-setup/roles/v1aX_integration_test/tasks/post_pivot.yml

```diff
@@ -1,71 +0,0 @@
-  - name: Define number of BMH's
-    set_fact:
-      NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
-
-  - name: Untaint all nodes.
-    shell: |
-        kubectl taint nodes --all node-role.kubernetes.io/master-
-    environment:
-      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    ignore_errors: yes
```

Comment:
You need these lines in the beginning of `node_reuse.yml` file, specifically before the:
`   - name: Scale worker down to 0 to start testing KubeadmControlPlane node reuse test scenario.` task. 
I went inside the VM while running the feature tests and saw BMO and Ironic Pod in `pending` state in target cluster. Untainting nodes helped BMO and ironic pods to become in `running` state and pass the step where it was failing previously.